.github/workflows/memory-leaks-check/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2019-2024 Intel Corporation. All Rights Reserved.
+cmake_minimum_required(VERSION 3.8)
+
+project(RealsenseExamplesMemoryLeaksCheck )
+
+# Save the command line compile commands in the build output
+set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
+if(BUILD_GRAPHICAL_EXAMPLES)
+	add_executable( start-camera-for-mem-leak start-camera-for-mem-leak.cpp ${IMGUI_SOURCES})
```

Comment:
Let's call it `mem-leak-test'
today we start the camera tomorrow we may add other stuff